File: auth-center/auth-center-infra/src/main/java/com/jmsoftware/maf/authcenter/permission/response/GetServicesInfoResponse.java
package com.jmsoftware.maf.authcenter.permission.response;

import com.jmsoftware.maf.common.domain.springbootstarter.HttpApiResourcesResponse;
import lombok.Data;

import java.util.List;

/**
 * Description: GetServicesInfoResponse, change description here.
 *
 * @author 钟俊（zhongjun）, email: zhongjun@toguide.cn, date: 12/28/2020 9:30 AM
 **/
@Data
public class GetServicesInfoResponse {
    private List<ServiceInfo> list;

    @Data
    public static class ServiceInfo{
        private String serviceId;
        private HttpApiResourcesResponse httpApiResources;
    }
}

File: auth-center/auth-center-infra/src/main/kotlin/com/jmsoftware/maf/authcenter/permission/response/GetServicesInfoResponse.kt
package com.jmsoftware.maf.authcenter.permission.response

import com.jmsoftware.maf.common.domain.springbootstarter.HttpApiResourcesResponse

/**
 * Description: GetServicesInfoResponse, change description here.
 *
 * @author Johnny Miller (锺俊), e-mail: johnnysviva@outlook.com, date: 4/11/22 6:08 PM
 */
class GetServicesInfoResponse {
    lateinit var list: List<ServiceInfo>

    class ServiceInfo {
        lateinit var serviceId: String
        lateinit var httpApiResources: HttpApiResourcesResponse
    }
}
